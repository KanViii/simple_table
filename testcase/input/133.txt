INSERT a number
ASSIGN a -17262.62