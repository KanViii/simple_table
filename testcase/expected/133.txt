Invalid: ASSIGN a -17262.62